CMakeLists.txt

```diff
@@ -102,3 +132,60 @@ target_link_libraries(ttt_brain
 install(DIRECTORY launch/
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
 )
+
+###########
+## Build ##
+###########
+
+include_directories(include
+    ${catkin_INCLUDE_DIRS}
+    ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+# Build a ttt_board_sensor library
+add_library(${PROJECT_NAME} src/board_sensor/tictactoe_utils.cpp)
+target_link_libraries(${PROJECT_NAME}
+    ${QT_LIBRARIES}
+    ${catkin_LIBRARIES}
+)
+
+add_executable(cells_delimitation include/ttt/board_cells_delimitation.h
+                                  src/board_sensor/board_cells_delimitation.cpp)
+target_link_libraries(cells_delimitation ${PROJECT_NAME} ${OpenCV_LIBS} ${QT_LIBRARIES} ${catkin_LIBRARIES})
+
+add_executable(show_board_cells src/board_sensor/show_board_cells.cpp)
+target_link_libraries(show_board_cells ${PROJECT_NAME} ${OpenCV_LIBS} ${QT_LIBRARIES} ${catkin_LIBRARIES})
+
+add_executable(hsv_range_finder src/board_sensor/hsv_range_finder.cpp)
+add_dependencies(hsv_range_finder ${catkin_EXPORTED_TARGETS})
+target_link_libraries(hsv_range_finder ${PROJECT_NAME} ${OpenCV_LIBS} ${catkin_LIBRARIES})
+
+add_executable(board_scheme src/board_sensor/board_scheme.cpp)
+add_dependencies(board_scheme ${catkin_EXPORTED_TARGETS})
+target_link_libraries(board_scheme ${OpenCV_LIBS} ${catkin_LIBRARIES})
+
+add_executable(board_state_sensing src/board_sensor/board_state_sensing.cpp)
+target_link_libraries(board_state_sensing ${PROJECT_NAME} ${OpenCV_LIBS} ${QT_LIBRARIES} ${catkin_LIBRARIES})
+
+add_executable(board_motion_detection include/ttt/board_motion_detection.h
+                                      src/board_sensor/board_motion_detection.cpp)
+target_link_libraries(board_motion_detection ${PROJECT_NAME} ${OpenCV_LIBS} ${catkin_LIBRARIES})
+
+#############
+## Install ##
+#############
+
+install(
+        TARGETS ${PROJECT_NAME}
+        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+        RUNTIME  DESTINATION bin  # This is for Windows
+)
+
+install(DIRECTORY include/ 
+        DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+)
+
+
+
+
+
```

Comment:
Now you have two different parts in which you build things, the first one belongs to the tictactoe package whereas the other one belongs to the `ttt_board_sensor` package. Try to merge the two as if it was a single package for real! It is only a matter of moving some instructions.